airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/output/OutputConsumer.kt

```diff
@@ -136,7 +178,102 @@ private class StdoutOutputConsumer : OutputConsumer {
         }
     }
 
+    override fun accept(record: AirbyteRecordMessage) {
```

Comment:
Nit: I think we use `accept` to mean "process incoming messages".  Perhaps this is `emit`?

Reply:
It's `accept` because this interface extends `java.util.function.Consumer` which is a standard java idiom.